.nengobones.yml

```diff
@@ -43,72 +43,40 @@ setup_py:
 
 setup_cfg:
   pytest:
-    addopts: []
+    addopts:
+      - "--cov"
+      - "--cov-append"
```

Comment:
Do these need to be defined statically here? It should be set automatically in the test script.